docs/Containers/Container-Tasks.md

```diff
@@ -62,50 +62,66 @@ the initial release for container tasks.
 
 3.  Greater configurability: When running container tasks, resources
 such as CPU and memory usage must be explicitly configured, unlike
-in the current distro model. Users are also responsible for
-picking the image to use, and all required software is downloaded
-during runtime. Furthermore, the container your task runs on is
-dedicated solely to that task and is never reused, so you are free
-to do whatever you want with it without needing to worry about
+in the current distro model. Users are also free to bring their own
+custom image to use, where only the software dependencies required for
+their particular workflow are downloaded during runtime. Furthermore, 
+the container your task runs on is dedicated solely to that task and is not 
+reused, so you are free to do whatever you want with it without needing to worry about
 leaving the environment in a messy state for the next task.
 
-# How to get started
-## Image lifecycle and self service
-
+## How To Get Started
+This section will walk you through the steps to get started with running container tasks in Evergreen,
+which are at a high level as follows:
+1. Create your Dockerfile in accordance with our existing [secure image
+   policy](https://docs.google.com/document/d/1MMePuL5YBjJQcNdtwzU2kMLPSsRLzDyE0rhTVkmXDqo/edit)
+2. Follow the image creation and management steps [here](Container-Tasks.md#creating-and-managing-your-container-images) to obtain a usable image URI
```

Comment:
nit: you can omit the `Container-Task.md` part of the link when the section title is part of the same file.